classes/classes/CoC.as

```diff
@@ -455,6 +455,7 @@ package classes
 			this.swamp = new Swamp(pregnancyProgress, output);
 			this.plains = new Plains(pregnancyProgress, output);
 			this.forest = new Forest(pregnancyProgress, output);
+			this.deepWoods = new DeepWoods(forest);
```

Comment:
mmh, sorry for being a bit nitpicky here, but shouldn't that read
```as3
this.deepWoods = new DeepWoods(this.forest);
```
then?

Reply:
Can do, to make it more consistent.

Reply:
Sure, but the line before that uses `this.forest`. May be a bit confusing then I guess.